scripts/lib/api/generateApiComponents.ts

```diff
@@ -230,7 +230,7 @@ function prepareAttributeProps(
   const attributeTypeHint = text
     .slice(Math.min(colonIndex + 1, equalIndex), equalIndex)
     .trim();
-  const attributeValue = text.slice(equalIndex, text.length).trim();
+  const attributeValue = text.slice(equalIndex + 1, text.length).trim();
```

Comment:
Comment would be helpful:

> We skip the first character to leave off the `=`.

But should this be +2 to skip the blank space?

Reply:
> But should this be +2 to skip the blank space?

Technically, yes. The catch is that we need to use a `trim()` to it anyway, so it has no difference. I think it's good to use +1 and only remove the `=,` just in case any attribute doesn't have the space for any weird circumstance like the other bugs in the historical versions.

> Comment would be helpful

I'll add a general comment for `attributeValue` and `attributeTypeHint` :+1: